milestone6.md

```diff
@@ -0,0 +1,23 @@
+# Milestone 6
+
+## Current UI Screenshots
+
+### Home Screenshots
+
+![Home Screen](/images/milestone6/home1.png)
+
+![Home Screen](/images/milestone6/home2.png)
+
+### Listings Screenshot
+
+![List Screen](/images/milestone6/listings-page.png)
+
+![Map View](/images/milestone6/map.gif)
+
+## Explanation of UI Changes and Data Visualizations
+
+In out last milestone, our Map was static only displaying the different neighborhoods outlined along with the listing next to it. For this milestone, we added interactions to the map so that upon hovering on a outlined neighborhood, the area that the user is hovering on will change color. Additionally, we added a new area at the top of the list view where the details for the hovered upon area are presented. These interactions add interactivity to our listings page and the manner in which we visualize our data. 
+
```

Comment:
You could also mention we made the `div` containing the neighborhoods scrollable so now users can scroll within that `div` and still see the map and currently hovered neighborhood rather than those things scrolling out of view.